### PR TITLE
[WIP] Usage docs

### PR DESCRIPTION
**Pull request by rtnpro**
This adds usage docs for atomicapp workflow. This will be helpful for users of atomicapp in general, be it be for using or testing.

### CODE REVIEW

**Comment by bexelbie** [comment]
Two suggestions:

1. Add some text at the beginning that explains who the target of the document is and what they will be able to do after following the steps.
2. Move the step-level text from overview to "Usage" and rename "Usage" ... make the Overview section more like a tl;dr and overview the process, without reference to specific methods.

**Comment by bexelbie** [comment]
This still seems to need some introduction to me.  Specifically, I think it should talk a bit about how, philosophically, the app works.  It seems there are 2-3 modes.  1) Install to disk; 2) run from container; 3) run from local files.  Each one needs to be explained so the user can better process the examples.

**Comment by dustymabe** [comment]
ping @rtnpro. Can you look at @bexelbie's suggestions?

**Comment by bexelbie** [comment]
@dustymabe changes have been made to this commit - @rtnpro please review too

**Comment by rtnpro** [comment]
@bexelbie LGTM :+1: I will add some docs for ``atomicapp stop``.

**Comment by bexelbie** [comment]
Can we merge this, pending STOP usage information?

**Comment by rtnpro** [comment]
Added docs for ``stop``. Good to merge :+1: 

@dustymabe ^^

**Comment by bexelbie** [comment]
@rtnpro what about:

A Nulecule application can be stopped using the `atomicapp stop` command. Stop uses the `answers.conf.gen` file created during installation/execution of the Nulecule application.

**Comment by rtnpro** [comment]
@bexelbie Fixed docs for ``stop``

**Comment by bexelbie** [comment]
@rtnpro can you put `atomicapp stop` and `answers.conf.gen` in single quotes for fixed width?

**Comment by rtnpro** [comment]
@bexelbie Fixed the above as well!

File: docs/usage.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+### Installing From a local path
+
+If a Nulecule application's metadata has already been extracted to a local directory, there is no installation operation to be performed.  Therefore, this is a `noop` command.  You cannot use the `--destination` command line option with this form of installation.
+
+    $ sudo atomicapp install some_projectatomic_helloapache
+    ...
+    Your application resides in some_project_helloapache
+    Please use this directory for managing your application
+
```
Comment (dustymabe) [inline docs/usage.md]: I'm not sure I understand this example. As you said, it does nothing. Should we even include it?

File: docs/usage.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+### Other Options
+
+`atomicapp run` supports the following optional arguments:
+
+- `-a` or `--answers`: By default, `atomicapp run` looks for the `answers.conf` file in the installation directory. This option overrides this behavior and allows you to specify another file that contains the answers data.
+- `-write-answers`: By default, `atomicapp run` writes the `answers.conf.gen` file to the installation directory. This option overrides this behavior and allows you to specify another file that should be used.
+- `--provider`: This option lets you override the default provider and the prpvider, if any, specified in `answers.conf` file.
+- `--ask`: This causes `atomicapp` to prompt the user for values for parameters that have default values and are not specified in ``answers`` file.
+- ``-destination``: Specifies the path to install Nulecule metadata to when running a Nulecule application directly from a container image.
```
Comment (dustymabe) [inline docs/usage.md]: --destination - needs another dash

**Comment by cdrage** [comment]
@rtnpro

With the docs refactor I did, I don't think is needed anymore correct?

**Comment by cdrage** [comment]
ping @rtnpro

**Comment by cdrage** [comment]
Hey @rtnpro could we close this now?

**Pull request closed without merging**